modules/ROOT/pages/ibm-ctg-connector.adoc

```diff
@@ -131,6 +129,24 @@ The IBM CTG connector supports the following four operations:
 |Execute with multiple channels |Invokes a CICS program with multiple channels and containers.
 |===
 
+== Connector Performance
+
+To define the pooling profile for the connector manually, access the Pooling Profile tab in the global element for the connector.
+
+For information on pooling, see xref:tuning-performance.adoc[Tuning Performance].
+
+== Enable Logging for Requests and Responses
+
+To enable logging of request and response interaction with the IBM CTG Connector, configure a logger in the Loggers tag in your Mule app's log4j2.xml file:
```

Comment:
logging for or logging of? Choose the most precise and use only it, unless you intend the distinction.